### DISCUSSION TITLE
Gopro 11 black storage the video in webcam mode

### DISCUSSION THREAD

**Discussion by Helloworld-kuang** [gopro/labs#315] [Q&A]
When gopro 11 black operate in webcam mode, could it storage the videos to SDcard at the same time?

**Comment by dnewman-gpsw** [comment]
I would like that too, but it is not a feature that the camera has.

**Reply by Helloworld-kuang** [reply]
Thank you for your answer, and if you know which mode can transfer the video data by USB and storage the videos to SDcard at the same time?

**Comment by cprima** [comment]
> When gopro 11 black operate in webcam mode, could it storage the videos to SDcard at the same time?

**When I had such a requirement** I went the "traditional broadcast route" and
- connected a GoPro with MediaMod clean HDMI output
- to a BlackMagic ATEM Mini Pro ISO (amongst other HDMI inputs)
- which records to an attached SSD drive
- while outputting via USB as a webcam to e.g. Zoom

The cost for this is about $1000 though. It does open the whole world of media production and from that angle the price is surprisingly affordable.
